airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_message.yaml

```diff
@@ -41,9 +41,8 @@ definitions:
         type: object
         existingJavaType: com.fasterxml.jackson.databind.JsonNode
       emitted_at:
-        description: "when the data was emitted from the source."
-        type: string
-        format: date-time
+        description: "when the data was emitted from the source. epoch in second."
```

Comment:
why epoch seconds versus millis? i think i don't have a big preference on which we use more that it would be nice if we were more consistent. i have generally been using millis elsewhere so far.

Reply:
can be millis